service/src/main/java/uk/gov/mca/beacons/api/beaconsearch/rest/BeaconSearchSpecification.java

```diff
@@ -68,14 +85,25 @@ > hasEmailOrRecoveryEmail(String email) {
       );
   }
 
-  public static @Nullable Specification<BeaconSearchEntity> hasAccountHolder(
-    UUID accountHolderId
+  public static @Nullable Specification<
+    BeaconSearchEntity
+  > isGreaterThanOrEqualTo(OffsetDateTime dateTime, String criteriaName) {
```

Comment:
These  methods, and the predicates below, seem to be about dates.

I think it would be good to make that clear in the method names. 

E.g. perhaps this one could be something like `hasDateOnAfter` or maybe  `hasDateGreaterThanOrEqualTo` depending on your preference.